docs/post/memcards.md

```diff
@@ -0,0 +1,124 @@
+
+Ross Ridge's original MyMC program works without issue on Windows 10 and newer and is a Windows-exclusive program. However, it predates the wide availability of PSV files – the format used to extract PS2 saves from a PS3 – and does not have support for these. For non-advanced users on Windows, this is probably the most straightforward version to run, as it is a basic Windows `.exe` file.
+
+### MyMC+ Python port by thestr4ng3r
+
+A Python 3 native port was created by thestr4ng3r which made some adjustments and added some additional features, most notably PSV support. While this version is cross-platform and more feature-complete, we generally feel it has been obsoleted due to some compatibility issues it had with Python libraries such as wxPython, and it has been improved upon by a newer release. We generally recommend not using this version, but it is important to remember it as a stepping stone for the latest revision.
+
+### MyMC++ Python port by Adubbz
+
+A second Python 3 native port is currently our recommended choice for users who understand how to follow more detailed setup instructions. This version brings in the improvements from MyMC+ such as cross-platform support and PSV support, adds additional usability improvements, and reliably runs on the latest versions of Python 3. This version also adds support for adding save files to the .mc2 memory card format used by the MemCard PRO2.
+
+## Using MyMC
+
+### Prerequisites
+
+[The Windows Visual C++ 2008 Runtime (x86) is required](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist#visual-studio-2008-vc-90-sp1-no-longer-supported)
+
+:::info
+The x64 version will not work; you must install the x86 version if you do not have it already.
+:::
+
+### Download MyMC
+
+MyMC is a portable program and does not "install". [You can find the original MyMC here](http://www.csclub.uwaterloo.ca:11068/mymc/)
+
+Download the `mymc-alpha-2.7.zip` file and extract it to a folder of your choice. Launch MyMC by running `mymc-gui.exe`.
+
+## Using MyMC++
+
+### Prerequisites
+
+Windows and MacOS users will need to install the [latest version of Python from here.](https://www.python.org/downloads/)
+
+:::caution
+While going through the installer, make sure to check the box to add Python to your PATH. You may need to reboot your PC after installing to ensure your PATH variable is set properly; more on this later.
+
+<Image src={require("./img/memcard_python_path.webp").default} />
+:::
+
+:::tip
+Most Linux distributions come bundled with Python 3 already installed. If your distribution does not include Python 3, you should try first to install it using your distribution's package manager.
+
+It is also recommended that you use `pipx` to install MyMC++ as recent Python version have implemented externally managed environment, which forces you to first manually setup a virtual environment before you can start installing MyMC++. `pipx` will handle all of the process automatically.
```

Comment:
"as recent python version have implemented externally managed environment"

Should have plurals:

"as recent python versions have implemented externally managed environments"

Also consider moving this down below to the actual download section which references `pip`.